.github/workflows/run-server-tests.yml

```
@@ -25,4 +25,4 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Run tests
-        run: make server-test
\ No newline at end of file
+        run: cd cli; make test
```

I think you meant `server`
```suggestion
        run: cd server; make test
```